pkg/minishift/docker/image/oci_image_handler.go

```diff
@@ -0,0 +1,366 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"bytes"
+	"encoding/json"
+	"github.com/briandowns/spinner"
```

Comment:
we rather not use the spinner, as this causes issues with the Console Plugin of DevStudio. Use the progress dots instead.

Reply:
Good catch. The original branch for this feature I created prior ProgressDots and I forgot to align. Will push an update.